multicluster/controllers/multicluster/common/helper.go

```diff
@@ -73,9 +73,23 @@ func FilterEndpointSubsets(subsets []corev1.EndpointSubset) []corev1.EndpointSub
 		}
 		if len(newAddresses) > 0 {
 			subset.Addresses = newAddresses
-			subset.Ports = s.Ports
+			subset.Ports = RemovePortName(s.Ports)
 			newSubsets = append(newSubsets, subset)
 		}
 	}
 	return newSubsets
 }
+
+// RemovePortName will remove each port's name since Antrea agent will skip reconciling
+// the Service/Endpoint if Service has no name but Endpoint has a name, vice versa.
```

Comment:
Both antrea-agent and kube-proxy behaves in this way by design because they use the name to associate port in service and endpoints. Otherwise it's impossible to handle multiple ports case. Why we don't keep the port name in service? Won't it support multiple-ports service in the future?

Reply:
Hi @tnqn, the main reason is we can't guarantee that the same services will have the same port name from different member clusters, so we didn't support it in multi-cluster.

Reply:
@jianjuns  do you think it's Ok to keep the name in Service and add a prerequisite in our user guide about all exported service's port should have the same name?

Reply:
I feel either strategy is fine. Ideally we should do translation when exporting. But it is no more an issue if we use ClusterIP or LB VIP to be the endpoints.